src/parsers.rs

```diff
@@ -31,7 +32,7 @@ fn take_digits(i: &[u8]) -> IResult<&[u8], u32> {
     let (i, digits) = take_while(is_digit)(i)?;
 
     if digits.is_empty() {
-        return Err(nom::Err::Error((i, nom::error::ErrorKind::Eof)));
+        return Err(Err::Error(Error::new(i, nom::error::ErrorKind::Eof)));
```

Comment:
Err Err Err Err Err.

Reply:
I hate that too, let's see if we can make that prettier